README.md

```diff
@@ -1,2 +1,80 @@
 # saleor-platform
 All Saleor services started from a single repository
+
+## Requirements
+1. [Docker](https://docs.docker.com/install/)
+2. [Docker Compose](https://docs.docker.com/compose/install/)
+
+
+## How to run it?
+
+### Mac OS
+1. Clone the repository `git clone git@github.com:mirumee/saleor-platform.git`
+2. Add the cloned `saleor-platform` directory into docker shared directories (Preferences -> Resources -> File sharing).
+3. Go inside cloned directory `cd saleor-platform`
+4. Fetch the submodules `git submodule update --init --recursive --jobs 3`
```

Comment:
Do we need `--recursive`?

Reply:
@patrys leftover, it is not, removed it.